playground/miden-wasm/src/expected_test_proof.rs

```diff
@@ -1,3 +1,2599 @@
 // quite a long proof, but it's just a test
```

Comment:
I ran `cargo fmt` and that got updated. I can check that file out of the commit if you want.